src/ua.c

```diff
@@ -816,6 +816,9 @@ static int create_register_clients(struct ua *ua)
 	if (uag_cfg() && str_isset(uag_cfg()->uuid))
 		add_extension(ua, "gruu");
 
+	if (uag_cfg() && uag_cfg()->sip_replaces_hdr)
+		add_extension(ua, "replaces");
```

Comment:
if this feature is supported by us, it is probably good to always have it enabled.

it is nice to have many config options, but too many can make it harder to use for average Joe :)